static/index.html

```diff
@@ -12,18 +12,14 @@
 
   <!-- MAIN DEVELOPMENT AREA -->
   <a href="#top" class="skiplink">Skip to main content</a>
-  <div class="bg-digital-red pt-5 pb-1">
-    <div class="cc">
-      <a class="logo text-white hocus:text-white text-20" href="https://stanford.edu">Stanford
+  <div class="cc bg-digital-red pt-5 pb-1">
+    <a class="logo text-white hocus:text-white text-20" href="https://stanford.edu">Stanford
         University</a>
-    </div>
   </div>
-  <div role="banner" class="rs-py-6 bg-gradient-to-b from-black to-plum text-white">
-    <div class="centered-container">
-      <h1 class="splash-text">Decanter 7 With Tailwind Test Page.</h1>
-      <p class="intro-text">This is a test page filled with common HTML elements to be used to provide visual
-        feedback whilst building CSS systems and frameworks.</p>
-    </div>
+  <div role="banner" class="cc rs-py-6 bg-gradient-to-b from-black to-plum text-white">
+    <h1 class="splash-text">Decanter 7 With Tailwind Test Page.</h1>
+    <p class="intro-text">This is a test page filled with common HTML elements to be used to provide visual
```

Comment:
I'm just removing extra divs that are not needed. Putting the `cc` class in the outermost container does the same thing.